### PR TITLE
Berry 0001

### PR DESCRIPTION
**Pull request by karthikvenkata**
Hi suresh anna we have done some work on g3 project .Please verify

### CODE REVIEW

File: g3web/WebContent/WEB-INF/web.xml [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
-        <filter-name>struts2</filter-name>
-        <url-pattern>/*</url-pattern>
-    </filter-mapping>
- 
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://java.sun.com/xml/ns/javaee"
+	xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
+	id="WebApp_ID" version="3.0">
+	<display-name>StrutsHello</display-name>
+<filter>
```
Comment (VenkataVege) [inline g3web/WebContent/WEB-INF/web.xml]: @karthikvenkata @pranaychand  format web.xml

File: g3web/WebContent/WEB-INF/web.xml [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
-    <filter-mapping>
-        <filter-name>struts2</filter-name>
-        <url-pattern>/*</url-pattern>
-    </filter-mapping>
- 
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://java.sun.com/xml/ns/javaee"
+	xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
+	id="WebApp_ID" version="3.0">
+	<display-name>StrutsHello</display-name>
```
Comment (VenkataVege) [inline g3web/WebContent/WEB-INF/web.xml]: @karthikvenkata @pranaychand  Display-name should we our project name,

File: g3web/WebContent/WEB-INF/web.xml [hunk 3]
```diff
@@ -1,23 +1,19 @@
...
+		<filter-name>struts2</filter-name>
+		<url-pattern>/*</url-pattern>
+	</filter-mapping>
 
-  
-  
+	<welcome-file-list>
+		<welcome-file>login.jsp</welcome-file>
+	</welcome-file-list>
 </web-app>
```
Comment (VenkataVege) [inline g3web/WebContent/WEB-INF/web.xml:18]: @karthikvenkata @pranaychand  Remove extra space.

File: g3web/WebContent/login.jsp [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<%@ taglib uri="/struts-tags" prefix="s"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<s:form action="hello">
```
Comment (VenkataVege) [inline g3web/WebContent/login.jsp]: @karthikvenkata @pranaychand Format this file.

Comment (VenkataVege) [inline g3web/WebContent/login.jsp]: @karthikvenkata @pranaychand  Action should not be hello. Make them close to real actions. For example this should be login.

File: g3web/WebContent/login.jsp [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<%@ taglib uri="/struts-tags" prefix="s"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<s:form action="hello">
+	<s:textfield name="uname" label="USERNAME"></s:textfield>
```
Comment (VenkataVege) [inline g3web/WebContent/login.jsp]: @karthikvenkata @pranaychand  do not use short cuts, rename uname to username.

File: g3web/WebContent/login.jsp [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<%@ taglib uri="/struts-tags" prefix="s"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<s:form action="hello">
+	<s:textfield name="uname" label="USERNAME"></s:textfield>
+	<s:password name="pwd" label="PASSWORD" />
```
Comment (VenkataVege) [inline g3web/WebContent/login.jsp]: @karthikvenkata @pranaychand  Same here, change it to password.

File: g3web/src/com/g3web/services/TestService.java [hunk 7]
```diff
@@ -1,5 +1,6 @@
 package com.g3web.services;
 
 public class TestService {
+	String s = "pranay";
```
Comment (VenkataVege) [inline g3web/src/com/g3web/services/TestService.java]: @karthikvenkata @pranaychand  This is just a test file. Please remove this.

File: g3web/src/struts.xml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE struts PUBLIC "-//Apache Software Foundation//DTD Struts  
+Configuration 2.1//EN" "http://struts.apache.org/dtds/struts-2.1.dtd">
+<struts>
+	<package name="o" extends="struts-default">
+		
+	</package>
+</struts>
```
Comment (VenkataVege) [inline g3web/src/struts.xml:7]: @karthikvenkata @pranaychand  Format this file as well.

File: g3web/src/struts.xml [hunk 9]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE struts PUBLIC "-//Apache Software Foundation//DTD Struts  
+Configuration 2.1//EN" "http://struts.apache.org/dtds/struts-2.1.dtd">
+<struts>
+	<package name="o" extends="struts-default">
```
Comment (VenkataVege) [inline g3web/src/struts.xml]: @karthikvenkata @pranaychand  name="o" ? Everything should have a meaning to it. Please rename "o"

**Comment by pranaychand** [comment]
Made some changes according to the comments. Please let me know if we need anything to be changed.

**Comment by VenkataVege** [comment]
LGTM :+1:

**Pull request merged**